shell/agents/Microsoft.Azure.Agent/Command.cs

```diff
@@ -117,17 +120,37 @@ private void ReplaceAction()
 
                     _values.Add(item.Name, value);
                     _agent.SaveUserValue(item.Name, value);
+                    DetailedMessage.Add(item.Name.Trim(new Char[] { '<', '>' }), true);
```

Comment:
Why trimming `<` and `>`? The placeholder format may change after the AzCLI team deploys their latest branch, so can you just record the placeholder as is, regardless of what format it uses?

Reply:
Because it will be encoded by JSON Serializer, and what in App Insight would be:
![image](https://github.com/user-attachments/assets/dcec8bff-360d-481a-947c-250cc7e0e206)

Reply:
Added back `<` and `>` by customizing the Encoder.